[2024-07-24T10:56:33.457+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-24T10:56:33.468+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.third_task manual__2024-07-24T10:54:30.841068+00:00 [queued]>
[2024-07-24T10:56:33.471+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.third_task manual__2024-07-24T10:54:30.841068+00:00 [queued]>
[2024-07-24T10:56:33.472+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 6
[2024-07-24T10:56:33.476+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): third_task> on 2024-07-24 10:54:30.841068+00:00
[2024-07-24T10:56:33.479+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=339) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-24T10:56:33.480+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'third_task', 'manual__2024-07-24T10:54:30.841068+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmppev1pwj3']
[2024-07-24T10:56:33.482+0000] {standard_task_runner.py:64} INFO - Started process 340 to run task
[2024-07-24T10:56:33.482+0000] {standard_task_runner.py:91} INFO - Job 16: Subtask third_task
[2024-07-24T10:56:33.524+0000] {task_command.py:426} INFO - Running <TaskInstance: first_dag.third_task manual__2024-07-24T10:54:30.841068+00:00 [running]> on host 312fcc3259a2
[2024-07-24T10:56:33.631+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Pattakit' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-07-24T10:54:30.841068+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-24T10:54:30.841068+00:00'
[2024-07-24T10:56:33.632+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-24T10:56:33.640+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-24T10:56:33.640+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey i am task three']
[2024-07-24T10:56:33.646+0000] {subprocess.py:86} INFO - Output:
[2024-07-24T10:56:33.648+0000] {subprocess.py:93} INFO - hey i am task three
[2024-07-24T10:56:33.648+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-24T10:56:33.649+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-24T10:56:33.658+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=third_task, run_id=manual__2024-07-24T10:54:30.841068+00:00, execution_date=20240724T105430, start_date=20240724T105633, end_date=20240724T105633
[2024-07-24T10:56:33.671+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-24T10:56:33.684+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-24T10:56:33.685+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
